freqtrade/data/entryexitanalysis.py

```diff
@@ -247,7 +306,10 @@ def process_entry_exit_reasons(config: Config):
 
                 print_results(res_df,
                               analysis_groups,
-                              indicator_list)
+                              indicator_list,
+                              rejected_signals=rej_df,
+                              to_csv=to_csv,
+                              csv_path=csv_path)
```

Comment:
Based on the fallback used for `csv_path` - csv_path can never be `None` - making huge parts of the downstream logic pointless.